pom.xml

```diff
@@ -2415,6 +2418,18 @@
                 <artifactId>netty-transport</artifactId>
                 <version>${netty.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-all</artifactId>
+                <version>${netty.version}</version>
```

Comment:
did you verify that nothing else was using a different version of netty before this change?